src/config_file.c

```diff
@@ -177,11 +177,19 @@ static int parse_next(FILE *fd)
 	} else if (!strncmp(s, "WHITELIST=", 10)) {
 		s += 10;
 		if (*s)
-			parse_list(&conf.cntrls_whitelist, s);
+			parse_list(&conf.cntrls_includelist, s);
 	} else if (!strncmp(s, "BLACKLIST=", 10)) {
 		s += 10;
 		if (*s)
-			parse_list(&conf.cntrls_blacklist, s);
+			parse_list(&conf.cntrls_excludelist, s);
+	} else if (!strncmp(s, "INCLUDELIST=", 12)) {
```

Comment:
Can we use:
``` C
#define INCLUDELIST "INCLUDELIST="
#define EXCLUDELIST "EXCLUDELIST="
```
and use `sizeof()` instead hardcoded values.
You can try to create small function, to avoid code duplication but it needs to be somehow incorporated with existing `else if` branches so it is not hard requirement. I give it to you.
```C
bool _parse_and_add_to_list(char *s, const char const *key, struct list *list)
{
    if (strncmp(s, key, sizeof(key) != 0)
     return false;

    s += sizeof(key);
    parse_list(list, s);
    return true;
}
```

We are editing this code, this is why I requested the change. You don't need to update entries above.
The main reason of the change is that hardcoding strings and the length is a known error prone practice.

The code in example is not tested and verified :)

Reply:
`sizeof` is at compile time.  As each of the keys have different lengths, the only way a single key passed would work is if we used `strlen`.  I think you avoid "unsafe" functions, so I have created a slightly different replacement.

```C
static bool _parse_and_add_to_list(char *s, const char *key, size_t key_len, struct list *list)
{
	if (strncmp(s, key, key_len != 0))
		return false;

	s += key_len;

	if (*s)
		parse_list(list, s);
	return true;
}
```
We can use some constants with something like:

```C
#define KEYLEN(key)   (sizeof((key))- 1)	/* Subtract 1 for string terminator */

#define ALLOWLIST "ALLOWLIST="
#define ALLOWLIST_LEN KEYLEN(ALLOWLIST)
#define EXCLUDELIST "EXCLUDELIST="
#define EXCLUDELIST_LEN KEYLEN(EXCLUDELIST)
```

Reply:
Please use same approach for legacy blacklist and whitelist parser and please put comment saying that they are only for backwards compatibility.

Reply:
Done